process/process_areas/architecture_design/guidance/architecture_process_reqs.rst

```diff
@@ -17,58 +17,45 @@
 Process Requirements
 ====================
 
-Architectural Structuring
--------------------------
+Architectural Model
+-------------------
 
-.. gd_req:: Hierarchical structure of architectural elements
-   :id: gd_req__arch__hierarchical_structure
+.. gd_req:: Architecture Modeling
+   :id: gd_req__arch__model
    :status: valid
    :complies: std_req__iso26262__support_6431, std_req__iso26262__support_6432
    :satisfies: wf__cr_mt_featarch
 
-   Architectural elements shall be hierarchically structured on two levels:
-
-   * Feature
-   * Component
-
-   Additionally there shall be a view for the *Top Level SW component* container.
-
-.. gd_req:: Architecture Views
-   :id: gd_req__arch__viewpoints
-   :status: valid
-   :complies: std_req__iso26262__support_6432, std_req__iso26262__software_742
-   :satisfies: wf__cr_mt_featarch
-
-   The architecture shall be shown with following views on each architectural level:
+   For architecture design a model based approach should be used. The model shall consist of different architectural elements.
 
-   * static view
-   * dynamic view
-   * interface view
-
-   An additional view only shall be created on module level.
-
-.. gd_req:: Architecture Modeling
-   :id: gd_req__arch__model
+.. gd_req:: Hierarchical structure of architectural elements
+   :id: gd_req__arch__hierarchical_structure
    :status: valid
    :complies: std_req__iso26262__support_6431, std_req__iso26262__support_6432
    :satisfies: wf__cr_mt_featarch
 
-   For architecture design a model based approach should be used.
+   The architectural elements shall be hierarchically structured on two levels:
 
-.. gd_req:: Structuring of the architectural building blocks
+   * Feature Level
+   * Component Level
+
+.. gd_req:: Structuring of the architectural elements
    :id: gd_req__arch__build_blocks
    :status: valid
    :complies: std_req__iso26262__support_6431, std_req__iso26262__support_6432
    :satisfies: wf__cr_mt_featarch
 
-   For modeling the viewpoints following elements shall be used:
+   Following architectural elements shall be defined on the respective hierarchical level:
+
+   * Feature Level
+     * Feature (feat_arc_sta)
+     * Logical Interface (logic_arc_int)
+     * Logical Interface Operation (logic_arc_int_op)
 
-   * Feature
-   * Module
-   * Component
-   * Logical Interfaces
-   * Interfaces
-   * Interface Operation
+   * Component Level
+     * Component (comp_arc_sta)
+     * Interface (comp_arc_int)
+     * Interface Operation (comp_arc_int_op)
```

Comment:
Just so I understand correctly: these 3 are in component-repositories?

So this is the "real component"?